package.json

```diff
@@ -24,7 +24,9 @@
     "object-to-class",
     "typescript-serializer"
   ],
-  "dependencies": {},
+  "dependencies": {
```

Comment:
Please remove this dependency and add this functionality as a local util function.

```ts
export function isPromise<T>(p: any): p is Promise<T> {
  return p !== null && typeof p === 'object' && typeof p.then === 'function';
}
```

Reply:
Yeah, okay.